build/helm/dc_website_v1/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for dc-website.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Website service config.
+website:
+  # The values of the doamins that should point to the DC website instance.
+  # The DNS should be configured to have an "A" record that points to
+  # the IP attached in the ingress section.
+  domains:
+  githash:
+
+  # Ingress allows the website servcice to be exposed to the outside of the cluster.
+  # Currently only external global ips are supported.
+  ingress:
+    enabled: true
+    annotations:
+      # Name of the existing GCP external global static ip.
+      # Please create an external global static ip first if it does not exist.
+      # Warning: regional ips will not work.
+      kubernetes.io/ingress.global-static-ip-name:
+
+  image:
+    repository: gcr.io/datcom-ci/datacommons-website
+    pullPolicy: Always
+    tag: "latest"
+
+  flask:
+    env: "private"
```

Comment:
should these be empty?

Reply:
Yes, sorry I missed quite a few when I was clearing the file. Most of these were intended to be empty. But for flask.env, would it be better to default to "private" or empty?

Reply:
Should default to "" so each env must set it explicitly